external_libraries/mpi_python/python_interface.cpp

```diff
@@ -50,6 +50,10 @@ PYBIND11_MODULE(mpipython, m)
     .def("reduce",&PythonMPI::reduce<int>)
     .def("allreduce",&PythonMPI::allreduce<double>)
     .def("allreduce",&PythonMPI::allreduce<int>)
+    .def("scatter", &PythonMPI::scatter<double>)
```

Comment:
`scatter` -> `scatter_double`

Reply:
this is called `scatter` intentionally, ideally one does not have to specify the data-type at all
For scatter however it is not possible due to technical reasons (at least for now)

I will try to change this when migrating it to the mpi-core